### PR TITLE
fix(web): `@next/font` warning

### LINKED ISSUE #558: Fix `@next/font` warning
## Description

> Your project has `@next/font` installed as a dependency, please use the built-in `next/font` instead. The `@next/font` package will be removed in Next.js 14. You can migrate by running `npx @next/codemod@latest built-in-next-font .`. Read more: https://nextjs.org/docs/messages/built-in-next-font

## Steps to reproduce

- Go to the `web` folder and run `yarn dev`
- The warning shows up in the terminal

## Screenshot

CleanShot 2023-03-21 at 02 36 52@2x

### PR DESCRIPTION
**Pull request by NiazMorshed2007**
Fix: This change resolves #558 by removing @next/font error

### CODE REVIEW

File: web/src/pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import '../styles/globals.css';
-import { Inter } from '@next/font/google';
+import { Inter } from "next/font/google";
```
Comment (bukinoshita) [inline web/src/pages/_app.tsx:2]:
```suggestion
import { Inter } from 'next/font/google';
```

**Comment by rfoel** [comment]
This PR introduced a bug

```
error - (sc_server)/node_modules/next/font/google/index.js (1:6) @ eval
error - Error: You tried to import `next/font/google`, did you mean `@next/font/google`?
Read more: https://nextjs.org/docs/basic-features/font-optimization
    at eval (webpack-internal:///(sc_server)/./node_modules/next/font/google/index.js:1:7)
    at (sc_server)/./node_modules/next/font/google/index.js (/Users/rafael/Developer/hack/email/.react-email/.next/server/app/page.js:1025:1)
    at __webpack_require__ (/Users/rafael/Developer/hack/email/.react-email/.next/server/webpack-runtime.js:33:42)
    at eval (webpack-internal:///(sc_server)/./src/app/layout.tsx:10:74)
    at (sc_server)/./src/app/layout.tsx (/Users/rafael/Developer/hack/email/.react-email/.next/server/app/page.js:1046:1)
    at Function.__webpack_require__ (/Users/rafael/Developer/hack/email/.react-email/.next/server/webpack-runtime.js:33:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.collectGenerateParams (/Users/rafael/Developer/hack/email/.react-email/node_modules/next/dist/build/utils.js:713:17)
    at async Object.loadStaticPaths (/Users/rafael/Developer/hack/email/.react-email/node_modules/next/dist/server/dev/static-paths-worker.js:46:32) {
  type: 'Error',
  page: '/'
}
null
```

**Comment by mengnans** [comment]
I am getting this error now 
image

**Comment by bukinoshita** [comment]
What's your next version?

**Comment by mengnans** [comment]
@bukinoshita the next version is 13.0.6, coming from .react-email/package.json
image
Thanks in advance.

**Pull request merged**